From 4929260470361a27e22a6218fbbcf4e2dce1b67d Mon Sep 17 00:00:00 2001
From: Michael Simacek <msimacek@redhat.com>
Date: Mon, 2 Oct 2017 17:33:05 +0300
Subject: [PATCH 13/18] Add missing transitive dependencies

---
 gradle/dependencies.gradle | 15 ++++++++++++---
 1 file changed, 12 insertions(+), 3 deletions(-)

diff --git a/gradle/dependencies.gradle b/gradle/dependencies.gradle
index 6e17a88bc31..262bcff571d 100755
--- a/gradle/dependencies.gradle
+++ b/gradle/dependencies.gradle
@@ -35,7 +35,10 @@ libraries.ant = dependencies.module('org.apache.ant:ant:1.9.6') {
 
 libraries.asm =  'org.ow2.asm:asm-debug-all:6.0_ALPHA'
 libraries.cglib = 'cglib:cglib:3.2.5'
-libraries.bndlib = dependencies.module('biz.aQute.bnd:biz.aQute.bndlib:3.4.0')
+libraries.bndlib = dependencies.module('biz.aQute.bnd:bndlib:3.4.0') {
+    dependency 'biz.aQute.bnd:aQute.libg:3.4.0@jar'
+    dependency 'biz.aQute.bnd:biz.aQute.bnd.annotation:3.4.0@jar'
+}
 libraries.commons_cli = 'commons-cli:commons-cli:1.2@jar'
 libraries.commons_io = dependencies.module(versions.commons_io)
 libraries.commons_lang = 'commons-lang:commons-lang:2.6@jar'
@@ -101,7 +104,9 @@ libraries += [
     nekohtml: 'net.sourceforge.nekohtml:nekohtml:1.9.14',
     xbean: 'org.apache.xbean:xbean-reflect:3.4@jar', //required by maven3 classes
     nativePlatform: 'net.rubygrapefruit:native-platform:0.14',
-    jansi: dependencies.module('org.fusesource.jansi:jansi:1.14'),
+    jansi: dependencies.module('org.fusesource.jansi:jansi:1.14') {
+        dependency "org.fusesource.jansi:jansi-native:1.14"
+    },
     xerces: "xerces:xercesImpl:2.9.1",
     objenesis: 'org.objenesis:objenesis:1.2@jar',
     jsoup:'org.jsoup:jsoup:1.6.3',
@@ -211,7 +216,11 @@ libraries.awsS3 = [
 ] + libraries.commons_httpclient + libraries.joda
 
 // keep in sync with ScalaLanguagePlugin code
-libraries.zinc = 'com.typesafe.zinc:zinc:0.3.15'
+libraries.zinc = dependencies.module('com.typesafe.zinc:zinc:0.3.15') {
+    dependency "org.scala-sbt:compile"
+    dependency "org.scala-sbt:compiler-integration"
+    dependency "org.scala-sbt:incremental-compiler"
+}
 
 libraries.gcs = [
         dependencies.create('com.google.apis:google-api-services-storage:v1-rev78-1.22.0') {
-- 
2.20.1

